examples/react-native/index.js

```diff
@@ -9,7 +9,7 @@ import { name as appName } from './app.json';
 import { setupStorybook } from './storybook';
 
 //TODO: replace with env var
-const initStorybook = true;
+const initStorybook = false;
```

Comment:
Do we want to commit this change?

Reply:
Yes, we need to stop using storybook and only test the actual Authenticator by default